app/components/quantity-widget.js

```diff
@@ -0,0 +1,50 @@
+import { computed } from "@ember/object";
+import Component from "@ember/component";
+
+export default Component.extend({
+  value: computed("model.package", function() {
+    if (this.get("package.requestedPackage")) {
+      return this.get("package.requestedPackage.quantity");
+    }
+    return 1;
+  }),
+  setValueIfValid(value, isValidValue, id) {
+    if (isValidValue) {
+      this.set("showErrorMessage", false);
+      Ember.$("#qty" + id).val(value);
```

Comment:
why `Ember.$("#qty" + id).val(value);`?

`this.set('value', value)` will this not work? 🤔

Reply:
It will but than i would no be able to call it in cart service.To avoid 2 way binding i have added in jquery way